src/main/frontend/src/components/Footer.module.css

```
@@ -0,0 +1,61 @@
+footer{
+    background-color: #252525;
+    font-family: Lato, Arial, serif;
+}
+
+h3{
```

space between classname and bracket

---

Also see comment about `li` class, it is better to have section specific class and inside of it you can have h3. ie
```suggestion
application_footer {
   h3 {
   }
}
```
this way we will avoid this class messing styles in other parts of application